e2e/machineset_migration_test.go

```diff
@@ -396,3 +1060,29 @@ func cleanupTestResources(ctx context.Context, cl client.Client, capiMachineSets
 		capiframework.DeleteAWSMachineTemplates(cl, template)
 	}
 }
+
+func verifyMAPIMachineSetHasCAPIMirror(cl client.Client, machineSetNameMAPI string) (*capiv1beta1.MachineSet, *capav1.AWSMachineTemplate) {
+	By("Check MAPI MachineSet has a CAPI MachineSet mirror")
+	var err error
+	var capiMachineSet *capiv1beta1.MachineSet
+	var awsMachineTemplate *capav1.AWSMachineTemplate
+
+	Eventually(func() error {
+		capiMachineSet, err = capiframework.GetMachineSet(cl, machineSetNameMAPI, capiframework.CAPINamespace)
+		return err
+	}, capiframework.WaitMedium, capiframework.RetryMedium).Should(Succeed(), "CAPI MachineSet should exist")
+
+	Eventually(func() error {
```

Comment:
This is duplicated 3x across these tests. We could helperfunc this, and just call that - making the tests easier to understand / read :)

Reply:
We could also log the error for ease of debugging in CI